index.html

```diff
@@ -1512,113 +1499,146 @@ <h1>WoT Architecture</h1>
+                and is the default Web representation of a <a>Thing</a>.
+                There MAY be other Web representations of a <a>Thing</a> such as an HTML-based user interface or simply an image of the physical entity.
+                However, the <a>WoT Thing Description</a> is a standardized, machine-understandable representation format
+                that allows <a>Consumers</a> to discover and interpret the capabilities of a Thing (through semantic annotations)
+                and to adapt to different implementations (e.g., different protocols or data structures) when interacting with a Thing,
+                thereby enabling interoperability across different <a>IoT platforms</a>, i.e., different ecosystems and standards.
+            </p>
+            <p class="ednote" title="TODO">
+                Simple figure to illustrate Consumer--Thing interaction.
+            </p>
+            <p>
+                A <a>Thing</a> can also be the abstraction of a virtual entity,
+                which is the composition of one or more Things (e.g., a room consisting of several sensors and actuators).
+                One option for the composition is to provide a single, consolidated <a>WoT Thing Description</a> that contains the superset of capabilities for the virtual entity.
+                In cases where the composition is rather complex, its <a>TD</a> may <emph>link</emph> to hierarchical sub-Things within the composition.
+                The main <a>TD</a> acts as entry point and only contain general metadata and potentially overarching capabilities.
+                This allows grouping of certain aspects of more complex Things.
+            </p>
             <p>
-                A <a>WoT Thing Description</a> (TD) contains information
-                about the structure, capabilities, functions and
-                interfaces of an exposed thing. Each Web of Things
-                device MUST have a corresponding TD. The thing
-                description is used by the consumer of a thing to
-                interact with it by obtaining and interpreting a TD of
-                the device. A TD contains, among other things, a
-                device’s identifier, functions and attributes of a
-                device and communication protocols (i.e. transport
-                layer) information. More information about the TD can be
-                found in <a href="#sec-thing-description"></a>.
+                Linking does not only apply to hierarchical Things, but relations between Things and other resources in general.
+                Link relation types express how Things relate, for instance, a switch controlling a light or a room monitored by a motion sensor.
+                Other resources related to a Thing can be manuals, catalogs for spare parts, CAD files, a graphical UI, or any other document on the Web.
+                Overall, Web linking among Things makes the Web of Things browsable.
```

Comment:
browsable -> navigable ?

Reply:
It's the **Web** of Things. Why dance to hide this? For me "browsable" also captures the usability we are used to from the Web.

Reply:
Clarified that machines can also browse